app.py

```diff
@@ -61,5 +62,58 @@ def get_shoes():
     conn.close()
     return jsonify(data)
 
+
+#post api to get the data
+
+@app.route('/addshoe',methods=['POST'])
+def add_shoe():
+
+    #Parse the json data
+    data = request.get_json()
+
+    #Extract data from jason
+    nike_data = data.get('nike')
+    adidas_data = data.get('adidas')
+    shoes_list_data = data.get('shoes_list')
+
+    #connect to the database
+    conn = get_db_connection()
+    cursor = conn.cursor()
+
+    try:
```

Comment:
This is wrong. Your expectation should be that the user will provide json data like this :
`{
"brand" : "nike", "name" : "jordan", "size" : 7, "price" : 2000
}`
Now you have to check for the brand in the request. Then insert the data into specific brand table. if you don't have that brand table return with some error. Also, after inserting the data into brand table. You have to insert one new row in the shoes_list with the foreign id of newly created row in brand table